rpcserver.go

```diff
@@ -512,6 +512,10 @@ func MainRPCServerPermissions() map[string][]bakery.Op {
 			Entity: "offchain",
 			Action: "write",
 		}},
+		"/lnrpc.Lightning/SendCustomMessage": {{
```

Comment:
Open questions if which sub-server this should live in 🤔

Reply:
Because there isn't an obvious sub-server for it to live in, I opted for the main one.